source/documentdb-support.txt

```diff
@@ -0,0 +1,84 @@
+.. _documentdb-support:
+
+===============================
+Amazon DocumentDB Compatibility
+===============================
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta:: 
+   :keywords: AWS, support
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+.. figure:: /figures/documentdb.png
+   :alt: Amazon DocumentDB logo
+
+`Amazon DocumentDB (with MongoDB compatibility) <https://docs.aws.amazon.com/documentdb/latest/developerguide/what-is.html>`__
+is a fast, reliable, and fully managed database service. Amazon DocumentDB makes it easy
+to set up, operate, and scale MongoDB-compatible databases in the cloud.
+With Amazon DocumentDB, you can run the same application code and use the same drivers
+and tools that you use with MongoDB.
+
+Version Information
+-------------------
+
+Amazon DocumentDB supports MongoDB v4.0 and MongoDB v5.0. These versions lack
+support for the newer features and functionality in the versions of MongoDB available
+on `MongoDB Atlas. <https://www.mongodb.com/docs/atlas/>`__
+
+.. tip:: MongoDB Version History
+   
+   To learn more about the features available in each version of MongoDB, see 
+   `MongoDB Evolved - Version History. <https://www.mongodb.com/evolved>`__
+
+Compatibility
+-------------
+
+As of November 2023, Amazon DocumentDB is only about 34 percent compatible with the
+MongoDB API.
+On Amazon DocumentDB, the following MongoDB features are
+available only in a limited capacity or are not supported at all:
+
+- Features: `Capped collections <https://www.mongodb.com/docs/manual/core/capped-collections/>`__,
```

Comment:
Suggestion: Bolding the list items might make this more legible.

```suggestion
- *Features*: `Capped collections <https://www.mongodb.com/docs/manual/core/capped-collections/>`__,
```

Reply:
yep, meant to do that, actually